sql/src/test/scala/org/apache/sedona/sql/functionTestScala.scala

```diff
@@ -459,6 +459,11 @@ class functionTestScala extends TestBaseScala with Matchers with GeometrySample
 
     }
 
+    it("Passed ST_NDims") {
```

Comment:
Please add more test cases for 2, 3, 4D (e.g., when Z and M present, or both present)